tools/ping/src/lib.rs

```diff
@@ -462,9 +496,11 @@ async fn ping_via_node(
                 let t = pending_timeout.unwrap();
                 app_info.pop_timeout(&t);
                 if let Some(csv) = latencies_csv.as_mut() {
+                    let account_id = app_info.peer_id_to_account_id(&t.peer_id);
+                    crate::metrics::PONG_TIMEOUTS.with_label_values(&[&peer_str(&t.peer_id, account_id)]).inc();
```

Comment:
this should go outside of the `if let Some(csv) = latencies_csv.as_mut() {...}`, since we still wanna update the metrics if we're not using the latencies CSV file